lib/infrastructure/hub_client/hub_client_demo.dart

```diff
@@ -81,62 +149,134 @@ class HubClientDemo {
 
     /// Guy Blinds
 
-    const String allRemoteCommandsBlinds =
-        '{"id":"96386cd0-419b-11ed-8246-e17a279f4d89","vendorUniqueId":"63b298","defaultName":"Guy Blinds","deviceStateGRPC":"ack","senderDeviceOs":"switcher","senderDeviceModel":"SwitcherDevicesTypes.switcherRunner","senderId":"94edb282-434a-11ed-bd96-f573a00b65aa","blindsSwitchState":"stop","deviceTypes":"blinds","compUuid":"34asdfrsd23gggg","deviceVendor":"switcherSmartHome","deviceDtoClass":"GenericBlindsDeviceDtos","stateMassage":"Hello World"}';
+    final GenericBlindsDE allRemoteCommandsBlinds = GenericBlindsDE(
+      uniqueId:
+          CoreUniqueId.fromUniqueString("96386cd0-419b-11ed-8246-e17a279f4d89"),
+      vendorUniqueId: VendorUniqueId.fromUniqueString("63b298"),
+      deviceVendor: DeviceVendor("switcherSmartHome"),
+      defaultName: DeviceDefaultName("Guy Blinds"),
+      deviceStateGRPC: DeviceState("ack"),
+      stateMassage: DeviceStateMassage("Hello World"),
+      senderDeviceOs: DeviceSenderDeviceOs("switcher"),
+      senderDeviceModel:
+          DeviceSenderDeviceModel("SwitcherDevicesTypes.switcherRunner"),
+      senderId: DeviceSenderId.fromUniqueString(
+          "94edb282-434a-11ed-bd96-f573a00b65aa"),
+      compUuid: DeviceCompUuid("34asdfrsd23gggg"),
+      blindsSwitchState: GenericBlindsSwitchState("stop"),
```

Comment:
This value should come from the device state veritable that exists in the .grpc file of the hub.

 `DeviceActions.`